byte RESULT_PASS = 0
byte RESULT_FAIL = 1
byte RESULT_HUMAN_REQUIRED = 2

string text_result
byte result
Plot[] plots
---
